tox.ini

```
@@ -26,11 +26,22 @@ commands = coverage erase
 
 [testenv:integrationtests]
 basepython = python3.8
-passenv = BRANCH
-commands =
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH BRANCH INTEGRATION_TESTS_REPO_URL TARGET_REPO
+allowlist_externals =
+	git
```

As in the other connected PRs:
```suggestion
    git
```
(To be checked throughout the PR)